.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,88 @@
+name: "Bug report 🐛"
+description: Report errors or unexpected behavior
```

Comment:
The name of this field is `about` not `description`

Reply:
this is a YML field defined by github, can't change.